src/routes/routes.tsx

```
@@ -0,0 +1,23 @@
+import { createBrowserRouter } from "react-router-dom";
+import LoginPage from "../pages/auth/LoginPage";
+import JoinPage from "../pages/auth/JoinPage";
+import Map from "../pages/map/Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)